resources/views/auth/register.blade.php

```diff
@@ -41,11 +41,11 @@
 
         <div class="flex items-center justify-end mt-4">
             <a class="underline text-sm text-gray-600 hover:text-gray-900 rounded-md focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500" href="{{ route('login') }}">
-                {{ __('Already registered?') }}
+                Bereits registriert?
```

Comment:
Ich würde mir eigentlich wünschen, dass wir sowohl DE wie auch EN anbieten können, das steht aber im Widerspruch zu Changes wie diesen. Ich ergänze die deutschen Strings mal in einem separaten PR via [Laravel Lang](https://laravel-lang.com/installation/).